content/millennium/dstu2/medications/medication-administration.md

```diff
@@ -9,13 +9,11 @@ title: MedicationAdministration | DSTU 2 API
 
 ## Overview
 
-The Medication Administration resource provides information about medications and vaccines administered to a patient or consumed by a patient including injections, intravenous solutions, and self administered oral medications.
-<br/><br/>
+The MedicationAdministration resource provides information about medications administered to a patient or consumed by a patient including injections, intravenous solutions, and self administered oral medications. 
+When using this resource, full administration details for vaccinations administered by the current healthcare system can be searched. For historically administered vaccines, or vaccines adminstered by other healthcare disciplines, care settings, or regions, the [`Immunization`] resource must be used for retrieving data.
 
-* **IMPORTANT NOTE**: Infuse or Bolus administration results for continuous infusion orders may require additional calculations to determine the actual amount of medication administered per ingredient. An additional calculation is required when the dosage quantity does not equal the contained Medication product ingredient amount denominator. In this case, calculate the ratio between the ingredient’s numerator divided by the denominator and multiply by dosage quantity. This will be the actual amount of medication administered.
-* **NOTE**:  These corrections may be breaking changes for your use cases.  Current users of the MedicationAdministration API should test in their non-production domains.  Cerner will coordinate with you and your developers of 3rd party applications to enable in your production domains.
+* **IMPORTANT NOTE**: Infuse or Bolus administration results for continuous infusion orders may require additional calculations to determine the actual amount of medication administered per ingredient. An additional calculation is required when the dosage dose does not equal the [contained] Medication ingredient strength denominator. In this case, calculate the ratio between the ingredient’s numerator divided by the denominator and multiply by dosage dose. This will be the actual amount of medication administered.
```

Comment:
second to last sentence, should it be dosage quantity?

Reply:
Yes.